sites/all/modules/hid/hid_profiles/hid_profiles.pages.inc

```diff
@@ -86,18 +86,16 @@ function hid_organizations_autocomplete($org_names, $extended = FALSE) {
   // Search for hr_organization taxonomy terms, where the term name or
   // organization field value match the given search string.
   $query = db_select('taxonomy_term_data', 't');
-  $query->leftJoin('field_data_field_acronym', 'fdfa', '(t.tid = fdfa.entity_id AND fdfa.entity_type = :fdfa_type)', array(':fdfa_type' => 'taxonomy_term'));
+  $query->leftJoin('field_data_field_acronym', 'fdfa', '(t.tid = fdfa.entity_id AND fdfa.entity_type = :fdfa_type AND fdfa.language = \'en\')', array(':fdfa_type' => 'taxonomy_term'));
```

Comment:
Would be great, but I don't think I will have time to do it, at least not soon, so unless you have time to do it, maybe we could merge this and change it to an EFQ later...